libcni/api.go

```diff
@@ -193,24 +207,77 @@ func (c *CNIConfig) getResultCacheFilePath(netName string, rt *RuntimeConf) stri
 	return filepath.Join(c.getCacheDir(rt), "results", fmt.Sprintf("%s-%s-%s", netName, rt.ContainerID, rt.IfName))
 }
 
-func (c *CNIConfig) setCachedResult(result types.Result, netName string, rt *RuntimeConf) error {
+func (c *CNIConfig) setCachedResult(result types.Result, config []byte, netName string, rt *RuntimeConf) error {
+
+	cached := cachedInfo{Config: config,
```

Comment:
@mccv1r0 newline here between the { and the Config:.  Also put the closing brace on a line by itself eg:

```
type foo struct {
    Member1: value,
    Member2: value2,
}
```